src/GovUk.Education.ExploreEducationStatistics.Common/Utils/BoundaryDataUtils.cs

```diff
@@ -0,0 +1,40 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace GovUk.Education.ExploreEducationStatistics.Common.Utils;
+public static class BoundaryDataUtils
+{
+    public static (string Name, string Code) GetFeatureDetails(IDictionary<string, object> properties)
+    {
+        // OBJECTID field appears to start being excluded from boundary files some time during 2022
+        var propertiesContainsObjectId = properties
+            .ElementAt(0)
+            .Key
+            .ToString()
+            .Equals("OBJECTID", StringComparison.InvariantCultureIgnoreCase);
+
+        var nameIndex = propertiesContainsObjectId ? 2 : 1;
+        var codeIndex = propertiesContainsObjectId ? 1 : 0;
+
+        var nameKey = properties.ElementAt(nameIndex).Key;
+        var codeKey = properties.ElementAt(codeIndex).Key;
+
+        if (!nameKey.EndsWith("NM", StringComparison.OrdinalIgnoreCase))
```

Comment:
I don't know if it's worth adding a comment for someone to understand why this is necessary. Same for the `CD` check.

Reply:
I'll refactor as per Ben's suggestion above, so comments shouldn't be necessary.